st2client/st2client/commands/keyvalue.py

```diff
@@ -168,6 +168,10 @@ def __init__(self, resource, *args, **kwargs):
         self.parser.add_argument('-e', '--encrypt', dest='secret',
                                  action='store_true',
                                  help='Encrypt value before saving.')
+        self.parser.add_argument('--pre-encrypted', dest='pre_encrypted',
```

Comment:
Why not just simply name the argument `--encrypted`?

Reply:
Yeah, I was thinking about it, but I thought it might be a bit confusing in combination with ``--encrypt`` argument.

I'n fine either way though.

Reply:
Adding mutually exclusive to the args will minimize the confusion.